### PR TITLE
update 3.4 and 3.5 changelogs

### PR DESCRIPTION
**Pull request by kkkkun**
update https://github.com/etcd-io/etcd/pull/16047 and https://github.com/etcd-io/etcd/pull/16048 to Changelog

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.4.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 <hr>
 
+## v3.4.27 (tbd)
+
+### etcd server
+- Fix [etcdserver: fix corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-3.4.md:10]: Nit - Should we remove the duplication given this is already under etcd server heading of and sentence starts with `Fix`?

```suggestion
- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```

File: CHANGELOG/CHANGELOG-3.5.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.10 (tbd)
 
+### etcd server
+- Fix [etcdserver: fix corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-3.5.md:10]: Nit - Should we remove the duplication given this is already under etcd server heading of and sentence starts with `Fix`?

```suggestion
- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
```

**Review by jmhbnz — COMMENTED** [review]
Thanks @kkkkun, just a minor suggestion around wording.

File: CHANGELOG/CHANGELOG-3.4.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 <hr>
 
+## v3.4.27 (tbd)
+
+### etcd server
+- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.4.md:10]:
```suggestion
- Fix [corruption check may get a `ErrCompacted` error when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```

File: CHANGELOG/CHANGELOG-3.5.md [hunk 4]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.10 (tbd)
 
+### etcd server
+- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.5.md:10]: same as above

**Comment by kkkkun** [comment]
> Thanks @kkkkun, just a minor suggestion around wording.

Thanks. Done.

File: CHANGELOG/CHANGELOG-3.5.md (hunk 4 above)
Comment (kkkkun) [reply CHANGELOG/CHANGELOG-3.5.md]: Done.

**Review by ahrtr — APPROVED** [review]
LGTM

Thanks @kkkkun

**Review by jmhbnz — APPROVED** [review]

**Review by serathius — APPROVED** [review]

**Comment by tjungblu** [comment]
@serathius can you hit the merge button please? then I can add my lease backport changes on top of it

**Pull request merged**